lib/administration/zone/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 3.12.0 FATAL_ERROR)
+# target_link_libraries for object targets
+
+foreach(variant IN ITEMS client server)
+  set(target_name "irods_zone_administration_${variant}")
+
+  add_library(
+    ${target_name}
+    OBJECT
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/zone_administration.cpp"
+  )
+  target_link_libraries(
+    ${target_name}
+    PRIVATE
+    nlohmann_json::nlohmann_json
+  )
+  target_include_directories(
+    ${target_name}
+    PUBLIC
+    "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+    PRIVATE
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_BINARY_DIR}/lib/core/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/lib/core/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/lib/api/include>"
```

Comment:
`$<BUILD_INTERFACE` include dirs should be `PUBLIC`

Reply:
Done.